tools/report-converter/codechecker_report_converter/infer/plist_converter.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from ..plist_converter import PlistConverter
+
+
+class InferPlistConverter(PlistConverter):
+    """ Infer plist converter. """
+
+    def _create_diag(self, message, files):
+        """ Creates a new plist diagnostic from the given message. """
+        diag = super(InferPlistConverter, self) \
+            ._create_diag(message, files)
+        diag['issue_hash_content_of_line_in_context'] = message.report_hash
```

Comment:
please use the context free hash the original hash could be kept in another tag in the plist report

Reply:
The original hash will be available in the `orig_issue_hash_content_of_line_in_context` tag.